GettingStarted.md

```diff
@@ -0,0 +1,214 @@
+# Table of Contents
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+
+- [Why Should I Use a Service Worker?](#why-should-i-use-a-service-worker)
+- [Terminology](#terminology)
+  - [Service Worker](#service-worker)
+  - [App Shell](#app-shell)
+  - [Dynamic Content](#dynamic-content)
+  - [Caching Strategy](#caching-strategy)
+- [Add `sw-precache` to Your Build](#add-sw-precache-to-your-build)
+  - [Automation](#automation)
+  - [Configuration](#configuration)
+    - [Basic](#basic)
+    - [Runtime Caching for Dynamic Content](#runtime-caching-for-dynamic-content)
+    - [Server-side Templating](#server-side-templating)
+    - [Fallback URL](#fallback-url)
+- [Examples](#examples)
+- [Other Resources](#other-resources)
+  - [Articles](#articles)
+  - [Videos](#videos)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Why Should I Use a Service Worker?
+
+You have a web app, and you'd like it to load quickly and work offline.
+
+You'd like to use proven tools to handle the details for you, to work around
+common gotchas and follow best practices.
+
+# Terminology
+
+## Service Worker
+
+A [service worker](http://www.html5rocks.com/en/tutorials/service-worker/introduction/)
+is a background script that intercepts network requests made by your web app.
+It can use the [Cache Storage API](https://developer.mozilla.org/en-US/docs/Web/API/Cache)
+to respond to those requests.
+
+You do not have to write your own service worker script; this guide will explain
+how to generate one customized for your web app.
+
+## App Shell
+
+"App Shell" refers to the local resources that your web app needs to load its
+basic structure. This will always include some HTML, and will likely also
+include CSS and JavaScript, either inline or in external files.
+
+Some static web apps consist entirely of an App Shell.
+
+A helpful analogy when thinking about your App Shell is to the code and
+resources that would be published to an App Store for a native iOS or Android
+application.
+
```

Comment:
"A helpful analogy is to think of your App Shell as the code and resources that would be published to an app store for a native iOS or Android application."

Suggested rewrite because I had a little trouble parsing the text as written.